playbooks/openshift-node/private/additional_config.yml

```diff
@@ -28,6 +32,13 @@
   vars:
     openshift_node_scale_up_group: "oo_nodes_use_flannel:oo_nodes_use_calico:oo_nodes_use_contiv:oo_nodes_use_kuryr"
 
+- name: Additional node config | custom_sdn
+  hosts: oo_nodes_use_custom_sdn
+  tasks:
+  - include_role:
+      name: '{{openshift_custom_sdn_role}}'
```

Comment:
I double quotes instead of single quotes are preferred here.  Also, please add spaces between curly braces and the variable name.

Reply:
This has been addressed.